Source/com/Administrator.cpp

```diff
@@ -294,7 +294,9 @@ namespace RPC {
                     }
                 } while ((loop->Decrement()) && (result == Core::ERROR_NONE));
 
+#ifdef __DEBUG__
```

Comment:
If this is doen to supress the ASSERT, this also requires a follow up inverstigation. This assert should not be triggetred.

Reply:
whyis this __DEBUG__ flag around this? ASSERT by definition  is only functional if "__DEBUG__" is set, so I see no added value here, or is it to avoid that the ASSERT fires? If so we need to further investigate